python/lsst/dbb/buffmngrs/handoff/local.py

```diff
@@ -27,6 +27,7 @@
 from datetime import datetime
 from .abcs import Command
 from .messages import FileMsg
+import re
```

Comment:
By convention (see [PEP8](https://pep8.org/#imports)), imports of Python standard libraries (like **re**) should go first. Also, please separate the imports of the standard libraries from the imports of the local modules by a blank line.